benchmarks/lasso/solvers/baseline.py

```diff
@@ -7,6 +7,12 @@
 class Solver(BaseSolver):
     name = 'Baseline'
 
+    parameters = {'use_acceleration': [False, True]}
+
+    def __init__(self, use_acceleration):
+        self.use_acceleration = use_acceleration
+        super().__init__()
```

Comment:
Ok here is an interesting point....

I did some "crazy" stuff so you don't need to implement an `__init__` for the solver. But you cannot implement it.
Basically, it makes it easier to ensure the solver is picklable and it helps exposing the solver parameters in his name. So anything that you set in parameters `key` will be accessible as `self.key`.

To me, the `__init__` should not do anything else than stocking parameters and the base class do that for you. But if you think it is too complicated, we can discuss it.

Reply:
This also makes sure that you have default parameters (the first of each list), which is why the test is failing.

Reply:
Ok, IMO we need to edit the doc with an example of solver with parameters because I took inspiration from the `simulated` dataset and there, there is an init.

Reply:
I agree, even I had forgotten about this, I had to look at the code.

Reply:
Actually, you can implement `__init__`, it just needs to have default values for all its parameters.

The fact that we don't implement it is to make sure we can report the parameters' value correctly but it might be tricky to enforce (don't feel very pythonic, default small value might be different to run test).
Something that would probably be easier is to use a decorator that calls `super().__init__` and log the parameters. This decorator would be used over the `__init__` function.